coalib/__init__.py

```diff
@@ -6,10 +6,60 @@
 
 
 import sys
+from os import chdir, getcwd
 from os.path import join, dirname
+from subprocess import check_output, CalledProcessError
 
 
-VERSION_FILE = join(dirname(__file__), 'VERSION')
+COALIB_DIR = join(dirname(__file__))
+
+VERSION_FILE = join(COALIB_DIR, 'VERSION')
+
+COMMIT_FILE = join(COALIB_DIR, 'COMMIT')
+
+
+def get_current_commit(git_dir):
+    '''
+    :param git_dir:     string containing the directory to check the
```

Comment:
'git_dir' is quite ambiguous.  It could be the directory where `git` is located.

I think this should be more like `git_repo_dir`.